src/index.js

```diff
@@ -27,4 +27,7 @@ module.exports = (chai, utils) => {
       new Assertion(sortDeep(a), msg).to.deep.equal(sortDeep(b));
     }
   });
+
+  chai.assert.deepEqualInAnyOrder = (actual, expected, message) => chai.expect(actual).to.deep.equalInAnyOrder(expected);
```

Comment:
don't we need to care about the message?

Reply:
oops!  I've added a PR for this at https://github.com/oprogramador/deep-equal-in-any-order/pull/71